pandas/io/parsers.py

```diff
@@ -101,42 +100,56 @@ class ParserWarning(Warning):
 keep_default_na : bool, default True
     If na_values are specified and keep_default_na is False the default NaN
     values are overridden, otherwise they're appended to
-parse_dates : boolean, list of ints or names, list of lists, or dict, default False
-    If True -> try parsing the index.
-    If [1, 2, 3] -> try parsing columns 1, 2, 3 each as a separate date column.
-    If [[1, 3]] -> combine columns 1 and 3 and parse as a single date column.
-    {'foo' : [1, 3]} -> parse columns 1, 3 as date and call result 'foo'
-    A fast-path exists for iso8601-formatted dates.
+parse_dates : various, default False
+    Acceptable input types
+    * boolean. If True -> try parsing the index.
+
+    * list of ints or names
+
+      * If [1, 2, 3] -> try parsing columns 1, 2, 3 each as a separate date
+        column.
+
+    * list of lists
+
+      * If [[1, 3]] -> combine columns 1 and 3 and parse as a single date
+        column.
+
+    * dict
+
+      * {'foo' : [1, 3]} -> parse columns 1, 3 as date and call result 'foo'
+
```

Comment:
Yes, as you say, in principle rst needs blank lines if you use a nested list (but I am not sure how this is actually handled in the docstrings, should test that). 
But in this case, as trade-off between looking nice in html and in plain text, I would maybe go without the nesting of the lists? (as now it takes quite some space in the plaintext version). Something like:

```
* boolean: If True -> try parsing the index.
* list of ints or names: If [1, 2, 3] -> try parsing columns 1, 2, 3 each as a separate date
  column.
* list of lists: If [[1, 3]] -> combine columns 1 and 3 and parse as a single date
  column.
* dict: {'foo' : [1, 3]} -> parse columns 1, 3 as date and call result 'foo'
```